src/style.css

```diff
@@ -1,3 +1,63 @@
+html {
+  margin: 0;
+  padding: 0;
+  --alternative-text-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  margin: 0;
+}
+
+.logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: block;
+  cursor: pointer;
+  color: black;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 23px 0;
+  margin-left: 20px;
```

Comment:
You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you want one element to be push off from another element use margins at THESE elements, not their children

Reply:
And pay attention that first child (nav__item) shouldn't have margin-left

Reply:
✅